### DISCUSSION TITLE
Rich text query preview / limit chars?

### DISCUSSION THREAD

**Discussion by cklaymen** [strapi/strapi#7840] [Legacy Topics]
Hi,

I am not sure if there is possibility to fetch only selected numbers of chars from rich text (graphql)? I would like to fetch eg. First 100 chars to list blog posts preview.

Thanks for advise

**Comment by derrickmehaffy** [comment]
We do not have a feature for that directly and it would require you to build a custom action to do so. See the graphql schema customization documentation here: https://strapi.io/documentation/v3.x/plugins/graphql.html#customize-the-graphql-schema
